cmd/root.go

```diff
@@ -51,20 +51,13 @@ func Execute() {
 	viper.BindPFlag("verbosity", pf.Lookup("verbosity"))
 	argToEnv["verbosity"] = "VERBOSITY"
 
-	client.CmdPush.Flags().StringVarP(&client.FlagServe, "serve", "s", "", "inference service to serve the model (kfserving, seldon_mlflow, seldon_sklearn, knative, deployment)")
-
 	config.AddEnvToUsage(rootCmd, argToEnv)
 
-	rootCmd.AddCommand(CmdCompletion)
 	rootCmd.AddCommand(client.CmdInstall)
 	rootCmd.AddCommand(client.CmdUninstall)
 	rootCmd.AddCommand(client.CmdInfo)
-	rootCmd.AddCommand(client.CmdOrgs)
-	rootCmd.AddCommand(client.CmdCreateOrg)
-	rootCmd.AddCommand(client.CmdPush)
 	rootCmd.AddCommand(client.CmdDeleteApp)
 	rootCmd.AddCommand(client.CmdApps)
```

Comment:
Are these still needed ? Why ?

Reply:
For sentimental value? I'm still using it when checking list of installed apps...

... but OK, I'll remove it too.

Reply:
As discussed elsewhere, we do need comparable support in the new fuseml CLI for listing and deleting "apps". Only in the context of composable workflows, we identified "apps" as the prediction services - or to be generic, any "services" - created by the workflow runs. It could be that we need https://github.com/fuseml/fuseml/issues/75 implemented to satisfy this requirement.